js/index.js

```diff
@@ -0,0 +1,71 @@
+import AwesomeBooks from './modules/books.js';
+import LocalStorage from './modules/local_storage.js';
+import CreateUI from './modules/create_ui.js';
+import CustomTime from './modules/time.js';
+
+const bookTitle = document.getElementById('title');
+const bookAuthor = document.getElementById('author');
+const form = document.getElementById('form');
+
+const localS = new LocalStorage();
+const ui = new CreateUI();
+
+function getVal(title, author, id) {
+  return new AwesomeBooks(title, author, id);
+}
+
+// important to hold length of local  storage
+let id = 0;
+
+function getIndexInLocalStorage() {
+  const books = localS.getLocalStorage();
+  // eslint-disable-next-line no-undef
+  return books.length;
+}
```

Comment:
- [ ] Please make sure to use ES6 arrow function syntax instead of the `function` keyboard.